WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -54,6 +58,26 @@
   </div>
 </div>
 
+<% if @display == 'admin' %>
+  <div class="form-group">
+    <%= label_tag(:status) %>
+    <div id="status" class="btn-group" data-toggle="buttons">
+      <label id="all" class="btn btn-info">
+        <%= radio_button_tag "status", "all" %>
+        <span class="caption">All</span>
+      </label>
+      <label id="warning" class="btn btn-warning">
+        <%= radio_button_tag "status", "warning" %>
+        <span class="caption">Warning</span>
+      </label>
+      <label id="danger" class="btn btn-danger">
+        <%= radio_button_tag "status", "danger" %>
+        <span class="caption">Danger</span>
+      </label>
+    </div>
+  </div>
+<% end %>
```

Comment:
We're just going to have to write some more javascript to show/hide this section of the competition searchbar when you change tabs. When you implement this, it might make sense to do in a generic way (maybe someday we'll have specific search stuff for the map view vs the list view vs the admin view. One clean way to do this would be to add a css class somewhere near the top that indicates what tab we're on, and then use css to hide/show sections of the competition search box.